tests/integration/chunkstore/test_chunkstore.py

```diff
@@ -537,7 +537,7 @@ def write_random_data(chunkstore_lib, name, month, days, securities, chunk_size=
     month: integer
     days: list of integers
     securities: list of integers
-    chunk_size: one of 'D', 'M', 'Y'
+    chunk_size: one of 'D', 'M', 'A'
```

Comment:
I guess we can check for 'Y' and convert to A, but pandas only takes 'A' now. Might be moot if i have to re-write the chunker anyways :)